bleak/backends/corebluetooth/PeripheralDelegate.py

```diff
@@ -10,6 +10,7 @@
 import itertools
 import logging
 from typing import Callable, Any, Dict, Iterable, NewType, Optional
+import async_timeout
```

Comment:
Since this isn't a builtin module, it would be nice to group it with the other 3rd party modules below.

Reply:
Adjusted in dfae35b8cba0908e36d1bdd2e3bae9ea21b04731